react/utils/index.ts

```diff
@@ -141,3 +141,22 @@ export function parseBankInvoiceUrl(url: string) {
     window.location.pathname + window.location.search
   )}`
 }
+
+export const getTotals = (totals: OrderItemTotal[]) => {
```

Comment:
Remember, preferer named functions instead of arrow ones...

Reply:
Done